File: Problems/Solved/771.js
describe("numJewelsInStones", () => {
    test("Testing", () => {
        let jewels = "z";
        let stones = "ZZ";
        expect(numJewelsInStones(jewels, stones)).toBe(0);
    });
});

/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function (jewels, stones) {
    let result = 0;
    for (let i = 0; i < stones.length; i++) {
        if (jewels.includes(stones[i])) result++;
    }
    return result;
};

File: Problems/771.rs
fn main() {}
#[cfg(test)]
mod tests {
    use super::*;
    #[test]
    fn test_case() {
        let jewels = "z".to_string();
        let stones = "ZZ".to_string();
        let expected_output = 0;
        assert_eq!(num_jewels_in_stones(jewels, stones), expected_output);
    }
}

pub fn num_jewels_in_stones(jewels: String, stones: String) -> i32 {
    let mut result = 0;
    for stone in stones.chars() {
        if jewels.contains(stone) {
            result += 1;
        }
    }
    result
}
